tx/textual/valuerenderer/coins.go

```diff
@@ -76,7 +81,116 @@ func (vr coinsValueRenderer) Format(ctx context.Context, v protoreflect.Value) (
 	}
 }
 
-func (vr coinsValueRenderer) Parse(_ context.Context, screens []Screen) (protoreflect.Value, error) {
-	// ref: https://github.com/cosmos/cosmos-sdk/issues/13153
-	panic("implement me, see #13153")
+func (vr coinsValueRenderer) Parse(ctx context.Context, screens []Screen) (protoreflect.Value, error) {
+	if len(screens) != 1 {
+		return protoreflect.Value{}, fmt.Errorf("expected single screen: %v", screens)
+	}
```

Comment:
Do we want to add tests for all these failure modes?

Reply:
We never considered Parsing as strict. If it can parse the Formatted screens, then it's good enough. For me it's not a hard requirement to have full coverage here.